src/moneypenny/config.py

```diff
@@ -40,3 +40,42 @@ class Configuration:
     Set by the ``DOCKER_SECRET_NAME`` environment variable,
     configured by the helm chart.
     """
+
+    config_dir: str = os.getenv(
+        "MONEYPENNY_CONFIG_DIR", "/opt/lsst/software/moneypenny/config"
+    )
+    """Directory where the Moneypenny ConfigMaps will be mounted.  Leave
+    at default in normal operation.
+    """
+
+    dossier_path: str = os.getenv("DOSSIER_FILE", "/opt/dossier")
+    """Path to the dossier file's mountpoint created in the initContainers.
+    Leave at default in normal operation.  Note that it does not contain the
+    filename (usually "dossier.json")
+    """
+
+    M: str = os.getenv("M_FILE", f"/{config_dir}/M/m.yaml")
```

Comment:
Nit: Single-letter all-caps variable names are normally reserved for type variables. Maybe `m_config`? Also, `M_CONFIG` seems like a better name here, since this is a configuration file specifically.

Reply:
fixed